### PR TITLE
chore(codeowners): add growth team as owners of telemetry definitions

### PR DESCRIPTION
**Pull request by bjoerge**
### Description

Adds growth as codeowners for `*.telemetry.ts` files.

### Notes for release

n/a

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -39,3 +39,6 @@
 # Shopify templates
 /packages/@sanity/cli/src/actions/init-project/templates/shopify* @thebiggianthead
 /packages/@sanity/cli/templates/shopify* @thebiggianthead
+
+# Telemetry definitions
+*.telemetry.ts @svirs
```
Comment (jtpetty) [inline .github/CODEOWNERS:44]: can we make this a team and not an individual?

Comment (bjoerge) [reply .github/CODEOWNERS]: Would be preferable, yeah. Either a separate team needs to be made, or the growth team needs to be made visible, see the error here: 89cc37649097b0f7cd1ba180b0395b35eb1ffca5

Comment (jtpetty) [reply .github/CODEOWNERS]: I added the growth team with `triage` role. Let's see if that fixes that error

Comment (svirs) [reply .github/CODEOWNERS]: Seeing this issue now:
```
make sure the team @sanity-io/growth exists, is publicly visible, and has write access to the repository
```

**Pull request merged**